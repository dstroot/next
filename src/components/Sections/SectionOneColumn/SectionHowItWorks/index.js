import React from 'react';

// components
import SectionOneColumn from '..';

// images

import arrows from '../SectionHowItWorks/media/arrows.svg';
import clock from '../SectionHowItWorks/media/clock.svg';
import pottedplant from '../SectionHowItWorks/media/pottedplant.svg';

const SectionHowItWorks = () => {
  return (
    <SectionOneColumn
      heading="How it works"
      content={
        <div className="row">
          <div className="col-md-4 align-bottom">
            <img
              className="mt-5"
              src={pottedplant}
              height={100}
              alt="Potted Plant"
            />
            <h2 className="nxt_heading-small mt-4">
              Start small
              <br /> today
            </h2>
            <p className="mt-4">
              Open an account for as little as $100. Set your contribution
              amount and we'll automatically debit your checking account every
              month.
            </p>
          </div>

          <div className="col-md-4">
            <img className="mt-5" src={arrows} alt="Arrows" height={100} />
            <h2 className="nxt_heading-small mt-4 ">
              Flex your <br /> contributions
            </h2>
            <p className="mt-4">
              Adjust or pause your contributions anytime as your financial
              picture changes. Get a bonus at work? Make a one-time contribution
              anytime.
            </p>
          </div>
          <div className="col-md-4">
            <img className="mt-5" src={clock} alt="Clock" height={100} />
            <h2 className="nxt_heading-small mt-4 ">
              Enjoy lifetime <br /> income
            </h2>
            <p className="mt-4">
              We stop your contributions 13 months before your elected start
              date. Once that day arrives, we'll pay you every month for the
              rest of your life.
            </p>
          </div>
        </div>
      }
    />
  );
};

export default SectionHowItWorks;
