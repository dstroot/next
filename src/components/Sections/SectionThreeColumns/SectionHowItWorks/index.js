import React from './node_modules/react';

// components
import SectionThreeColumns from '..';

// images
import Arrows from '../SectionHowItWorks/media/Arrows.svg';
import clock from '../SectionHowItWorks/media/clock.svg';
import pottedplant from '../SectionHowItWorks/media/pottedplant.svg';

const SectionHowItWorks = () => {
  return (
    <div className="py-6">
      <SectionThreeColumns
        heading="How it works"
        styleColumnAlign="align-items-center"
        contentLeft={
          <img
            className="mt-5"
            src={pottedplant}
            width={60}
            alt="Potted Plant"
          />
        }
        contentMiddle={
          <img className="mt-5" src={Arrows} width={115} alt="Arrows" />
        }
        contentRight={
          <img className="mt-5" src={clock} width={115} alt="Clock" />
        }
      />
      <SectionThreeColumns
        contentLeft={
          <h2 className="nxt_heading-small px-5">Start small today</h2>
        }
        contentMiddle={
          <h2 className="nxt_heading-small px-4">Flex your contributions</h2>
        }
        contentRight={
          <h2 className="nxt_heading-small px-4">Enjoy lifetime income</h2>
        }
      />
      <SectionThreeColumns
        contentLeft={
          <p className="px-4">
            Open an account for as little as $100. Set your contribution amount
            and we'll automatically debit your checking account every month.
          </p>
        }
        contentMiddle={
          <p className="px-4">
            Adjust or pause your contributions anytime as your financial picture
            changes. Get a bonus at work? Make a one-time contribution anytime.
          </p>
        }
        contentRight={
          <p className="px-4">
            We stop your contributions 13 months before your elected start date.
            Once that day arrives, we'll pay you every month for the rest of
            your life.
          </p>
        }
      />
    </div>
  );
};

export default SectionHowItWorks;
