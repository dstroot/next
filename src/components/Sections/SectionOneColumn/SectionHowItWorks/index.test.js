import React from 'react';
import { create } from 'react-test-renderer';
import { mount } from 'enzyme';
import SectionHowItWorks from '../SectionHowItWorks';

describe('SectionHowItWorks', () => {
  // mount the component
  let mountedComponent;
  const getMountedComponent = () => {
    if (!mountedComponent) {
      mountedComponent = mount(<SectionHowItWorks />);
    }
    return mountedComponent;
  };

  beforeEach(() => {
    mountedComponent = undefined;
  });

  it('it should render', () => {
    let tree = create(<SectionHowItWorks />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
