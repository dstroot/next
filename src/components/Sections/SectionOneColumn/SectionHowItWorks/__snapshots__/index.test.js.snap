// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHowItWorks it should render 1`] = `
<div
  className="py-6 undefined"
>
  <div
    className="container text-center"
  >
    <div
      className="react-reveal nxt_heading-large mb-5"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      How it works
    </div>
    <div
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <div
        className="row"
      >
        <div
          className="col-md-4 align-bottom"
        >
          <img
            alt="Potted Plant"
            className="mt-5"
            height={100}
            src="pottedplant.svg"
          />
          <h2
            className="nxt_heading-small mt-4"
          >
            Start small
            <br />
             today
          </h2>
          <p
            className="mt-4"
          >
            Open an account for as little as $100. Set your contribution amount and we'll automatically debit your checking account every month.
          </p>
        </div>
        <div
          className="col-md-4"
        >
          <img
            alt="Arrows"
            className="mt-5"
            height={100}
            src="arrows.svg"
          />
          <h2
            className="nxt_heading-small mt-4 "
          >
            Flex your 
            <br />
             contributions
          </h2>
          <p
            className="mt-4"
          >
            Adjust or pause your contributions anytime as your financial picture changes. Get a bonus at work? Make a one-time contribution anytime.
          </p>
        </div>
        <div
          className="col-md-4"
        >
          <img
            alt="Clock"
            className="mt-5"
            height={100}
            src="clock.svg"
          />
          <h2
            className="nxt_heading-small mt-4 "
          >
            Enjoy lifetime 
            <br />
             income
          </h2>
          <p
            className="mt-4"
          >
            We stop your contributions 13 months before your elected start date. Once that day arrives, we'll pay you every month for the rest of your life.
          </p>
        </div>
      </div>
    </div>
    <div />
  </div>
</div>
`;
